app/views/courses/course.json.jbuilder

```diff
@@ -27,6 +27,7 @@ json.course do
   json.published CampaignsCourses.exists?(course_id: @course.id)
   json.closed @course.closed?
   json.enroll_url "#{request.base_url}#{course_slug_path(@course.slug)}/enroll/"
+  json.wiki_prefix @course.home_wiki.project == 'wikidata' ? 'articles.items' : 'articles.pages'
```

Comment:
I think it makes the most sense to make `string_prefix` a method on the `Wiki` model, in parallel to the `string_prefix` method of `Course`. This way, we can use the same logic for server-rendered i18n messages as well, and easily include the prefix data in other endpoints besides course.json.

(Then you'd do something like `json.wiki_string_prefix @course.home_wiki.string_prefix` here.)